.gitignore

```diff
@@ -8,6 +8,7 @@ packages
 paket.dependencies
 paket.dependencies.locked
 .vs
+_ReSharper.Caches
```

Comment:
I prefer having ReSharper caches in solution folders. That way, I can safely delete them anytime I want, before they consume dozens of GBs.